components/views/services/dummydata.tsx

```diff
@@ -0,0 +1,52 @@
+const ServiceData = [
```

Comment:
FYI, the schema for the services is as follows:
```
services: {
    pageH1: String,
    pageH2: String,
    description: String,
    offerings: [
        {
            icon: String,
            title: String,
            description: String,
            bullets: [String]
        }
    ]
}
```

So, you can rework the dummy data, and the page construction to accommodate the schema 😄